client/components/data/domain-management/index.jsx

```diff
@@ -37,13 +35,11 @@ class DomainManagementData extends Component {
 	};
 
 	render() {
-		const { needsContactDetails, needsDomains, needsPlans, needsProductsList, selectedSite } =
-			this.props;
+		const { needsDomains, needsPlans, needsProductsList, selectedSite } = this.props;
 
 		return (
 			<div>
 				<PageViewTracker path={ this.props.analyticsPath } title={ this.props.analyticsTitle } />
-				{ selectedSite && needsContactDetails && <QueryContactDetailsCache /> }
```

Comment:
From what I can tell, this was never used. There are no selectors for the domain contact details in the domain-management pages; they use the Whois data instead.